schema/schema.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE music_albums(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  on_spotify BOOLEAN,
+  FOREIGN KEY(id) REFERENCES item(id)
```

Comment:
The foreign key is the one that relates to tables so instead of using 'references item(id)' it should be 'references geners(id)'

Reply:
hello, this has been solved.